cmd/kueue/main.go

```diff
@@ -155,6 +155,16 @@ func main() {
 	}
 	options.Metrics = metricsServerOptions
 
+	if features.Enabled(features.LocalQueueMetrics) {
+		if cfg.Metrics.LocalQueueMetrics != nil {
```

Comment:
```suggestion
		if cfg.Metrics.LocalQueueMetrics != nil && cfg.Metrics.LocalQueueMetrics.Enable {
```

What about this conditional?